SwiftParcel/d-docker-compose/micro-services-local.yml

```diff
@@ -2,12 +2,23 @@ version: "2.1"
 
 services:
 
+ api-gateway:
+    build: ../../SwifttParcel.API.Gateway
+    container_name: api-gateway
+    restart: unless-stopped
+    environment:
+      - NTRADA_CONFIG=ntrada.docker # TODO: Add this file
+    ports:
+      - 5000:80
+    networks:
+      - pacco
+
   identity-service:
     build: ../../SwifttParcel.Services.Identity
     container_name: apa/identity-service
     restart: unless-stopped
```

Comment:
Idk why some container names have this apa prefix and some don't. Also my docker doesn't like slashes in name.

Reply:
apa is hypothetical name of the dockerhub repo owner. The micro-servirces-local are to build the existing Dockerfiles. Services  are to build it from the  images from the dockerhub.